tests/langchain/test_langchain_autolog.py

```diff
@@ -187,9 +189,9 @@ def test_autolog_manage_run():
     assert len(traces) == 2
     for trace in traces:
         span = trace.data.spans[0]
-        assert span.span_type == "CHAIN"
-        assert span.inputs == {"product": "MLflow"}
-        assert span.outputs == {"text": TEST_CONTENT}
+        assert span.attributes[SpanAttributeKey.SPAN_TYPE] == "CHAIN"
+        assert json.loads(span.attributes[SpanAttributeKey.INPUTS]) == {"product": "MLflow"}
```

Comment:
note: After the entity unification follow-up, these will have `inputs` / `outputs` property so users don't need to deserialize by themselves.